specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -6810,6 +6810,78 @@
           "description": "The subscription name.",
           "type": "string",
           "readOnly": true
+        },
+        "azureCreditApplied": {
+          "description": "The azure credit applied.",
+          "readOnly": true,
+          "$ref": "#/definitions/Amount"
+        },
+        "billingCurrency": {
+          "description": "The billing currency.",
```

Comment:
Does it follow ISO 4217 currency code standard? If so, can you mention it in the description?

Reply:
Sure, Done.